apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/helpers.php

```diff
@@ -38,23 +35,12 @@ function dangerously_load_full_site_editing_files() {
  * @returns bool True if FSE is active, false otherwise.
  */
 function is_full_site_editing_active() {
-	/**
-	 * There are times when this function is called from the WordPress.com public
-	 * API context. In this case, we need to switch to the correct blog so that
-	 * the functions reference the correct blog context.
-	 */
-	$multisite_id  = apply_filters( 'a8c_fse_get_multisite_id', false );
-	$should_switch = is_multisite() && $multisite_id;
-	if ( $should_switch ) {
-		switch_to_blog( $multisite_id );
+	// We will always return false in admin and REST API contexts as we work towards getting rid of this.
+	if ( is_admin() || ( defined( 'REST_API_REQUEST' ) && REST_API_REQUEST ) ) {
+		return false;
 	}
 
-	$is_active = is_site_eligible_for_full_site_editing() && is_theme_supported() && did_insert_template_parts();
-
-	if ( $should_switch ) {
-		restore_current_blog();
-	}
-	return $is_active;
+	return is_theme_supported() && did_insert_template_parts();
```

Comment:
>is there a reason to get rid of the is_site_eligible_for_full_site_editing() check?

Oh well, it seems like we removed the function so... yeah 🤣

Reply:
We just didn't need it and I'm simplifying as much as possible. `did_insert_template_parts` tells us what we need to know for rendering on the frontend.

Reply:
There could be some edge cases I can't imagine, by definition. I was pretty excited to remove eligibility as a whole from the codebase but since we're keeping parts of this running, it would indeed be best to just keep things relatively undisturbed. This means we also need to maintain the filter on the backend of wpcom, which I think I'm currently removing with aforementioned intent in mind.